kafka/producer/base.py

```diff
@@ -27,6 +27,9 @@
 
 STOP_ASYNC_PRODUCER = -1
 
+# unlimited
+ASYNC_QUEUE_MAXSIZE = 0
```

Comment:
@bcwalrus a large limit would be same as unlimited, isn't it? The python queue handles ``zero`` as unlimited.